bytecode/bytecode.py

```diff
@@ -97,24 +97,27 @@ def __eq__(self, other):
 
 class Bytecode(_InstrList, BaseBytecode):
 
-    def __init__(self, instructions=None):
+    def __init__(self, instructions=[]):
```

Comment:
Hi @eric-wieser could you use a tuple or None, and check if the value is None or an iterable and use an iterable in ``__init__`` ?